### DISCUSSION TITLE
Unable to deploy the IndexOptimize script in Azure Sql server

### DISCUSSION THREAD

**Discussion by rajeshpalo2003** [olahallengren/sql-server-maintenance-solution#441] [Questions & Answers]
Unable to deploy the IndexOptimize script in Azure sql server which is managed by Microsoft. I'm getting below error.

Is there any other approach to deploy? Or do we need to deploy this in individual Azure SQL Databases?

**Msg 262, Level 14, State 1, Line 9
CREATE TABLE permission denied in database 'master'.**

**Comment by olahallengren** [comment]
It is not supported to create objects in the master database, in Azure SQL Database.

You need to create the stored procedures, in the individual user databases.

**Comment by Strada-EricFr** [comment]
But if you have a Azure SQL using an Elastic Pool with multiple database in it, you need to run the script: [IndexOptimize.sql](https://ola.hallengren.com/scripts/IndexOptimize.sql) in all these database ?
